script/setup

```diff
@@ -23,6 +23,9 @@ fi
 
 script/bootstrap
 
+# Avoid unsafe git error
+git config --global --add safe.directory "$PWD"
```

Comment:
If this is ran outside of a dev container, this will change a user's global git configuration. That sounds like something we don't want to do.

As a user myself, I would find that unacceptable behavior.

Reply:
I did not take into account that this script would be run outside devcontainer use.
Updated to only run when `DEVCONTAINER` env is set.